app/package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "headlamp",
-  "version": "0.29.0",
```

Comment:
Could you please remove the app/package-lock.json change? It’s unrelated to this PR.